geoips/plugins/modules/readers/abi_netcdf.py

```diff
@@ -269,6 +269,9 @@ def _get_variable_metadata(df):
             metadata[name] = df.variables[name][...]
             if metadata[name].size == 1:
                 metadata[name] = metadata[name][()]
+            if name in ["x", "y"]:
+                metadata[f"{name}_add_offset"] = df.variables[name].add_offset
+                metadata[f"{name}_scale_factor"] = df.variables[name].scale_factor
```

Comment:
Are these used? @evrose54

Reply:
These are used for an 'octopy' plugin package I'm working on. I need to have the add offset and scale factor to convert from ``i,j`` space to ``x,y`` space, and eventually to ``lat, lon`` space. Might be a good idea to add this to our standards and other readers.

Reply:
I think you should probably create an issue to finalize xarray standards / reader metadata / formatting, then link to this entire PR for reference.  Link to the issue from this conversation then you can resolve.

Reply:
See #817